### PR TITLE
My sisters hair- Project-business-site

### PR DESCRIPTION
**Pull request by Snabbina**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,274 @@
...
+
+/* syns i mobil vy */
+
+  @media (max-width: 767px){
+  
+    .top-section-container{
+      background-size: 100% 100%;
+      width: 100%;
+      height: 100%;
+    }
```
Comment (bealun) [inline code/style.css:208]: This need a peak since it's not responsive - the image pushes together and this need a solution before you're approved!

**Review by bealun — CHANGES_REQUESTED** [review]
Good job and a really cool layout for your site! 🌟 A few thing I want you to remember is to commit code a lot more ofter here on GitHub and also be consistent with the indentation - you will benefit from this! Other than that you've done an amazing job on the JavaScript by keeping it clean, great job with the validation and a lot of nice styling features. Be proud of yourself, it looks so good 🎉
I'm requesting some changes since the hero image isn't responsive and there's some unnecessary code and empty spaces that you need to remove before you're approved. Check out my comments!

**Comment by Snabbina** [comment]
Hey, thank you so much for the feedback!!!

I have  done the changes no, how do I get them to Git again? I have done
them locally in my code for the project so I dont seem to be able to
push them to my Git master anymore? Can you help me explaining how I should
do or refer my to some guide for this =)

best of the best!


/Sabina

Den fre 11 sep. 2020 kl 14:10 skrev Beatrice Lundevall <
notifications@github.com>:

File: code/index.html [hunk 2]
```diff
@@ -1,30 +1,197 @@
...
   <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta charset="utf-8">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1.0" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Overpass|Permanent+Marker" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/index.html:8]: Put in one now =)

File: code/index.html [hunk 3]
```diff
@@ -1,30 +1,197 @@
...
+    </div>
+    <div class="form-group">
+        <label for="lastName">Last name</label>
+        <input
+            id="lastName"
+            type="text"
+            placeholder="Type your last name"
+            name="lastNameValue"
+            required
+        />
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/index.html:51]: Thanks!

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,274 @@
...
+
+.form-submit {
+  width: 100px;
+  position: relative;
+} 
 
 .active + .answer {
-  /* some styling here */
+  display: block;
+}
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/style.css:90]: will do so!

File: code/index.html [hunk 5]
```diff
@@ -1,30 +1,197 @@
...
+      
     </section>
+
+  </main>
+
+<div class="footer">
+
+<p>Created by Sabina de Maré @ Technigo bootcamp 2020</p>
+
+</footer>
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/index.html]: changed it!

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,274 @@
...
+  align-items: center;
+  justify-content: center;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+  position: absolute; 
+  background-image: url("./Mysister.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/style.css:26]: yay!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,274 @@
...
+  justify-content: flex-start; 
+  flex-wrap: wrap; 
+  width: 30%;
+  } 
+
+  .accordion {
+  display: flex; 
+  justify-content: flex-start; 
+  flex-wrap: wrap; 
+  width: 30%;
```
[the remark this answers is not in the document]
Comment (Snabbina) [reply code/style.css]: that fixed it.

File: code/style.css (hunk 1 above)
Comment (Snabbina) [reply code/style.css]: Yes, changed the background-size to cover and changed the font-size to a smaller one in mobile view to make it more responsive, hope that dose it =)

Comment (Snabbina) [reply code/style.css]: I made all my code changes in my code (in the VS-editor) and pushed them again to Git, is that enough or should I do something else/more? plz let me know if so.

**Review by puj — APPROVED** [review]
Looks good! Nice updates!

**Pull request closed without merging**